src/ticks/syms.c

```diff
@@ -352,7 +352,7 @@ int address_is_code(int addr)
 
     while ( sect != NULL ) {
         if ( addr >= sect->start && addr < sect->end) {
-            return (strncmp(sect->name, "code_",5) == 0 || strncmp(sect->name, "smc_",4) == 0 );
+            return (strncasecmp(sect->name, "code_",5) == 0 || strncasecmp(sect->name, "smc_",4) == 0 );
```

Comment:
Ignore case when searching for section names.

Reply:
Afk. But I think there’s a #define that deals with windows portability.

Reply:
I found the code below in ticks.c
```
#if defined(_WIN32) || defined(WIN32)
#ifndef strcasecmp
#define strcasecmp(a,b) stricmp(a,b)
#endif
#endif
```
I can add something similar to syms.c to handle `strncasecmp()`

Reply:
Please bump it up to ticks.h which is included by both I believe.